Isolated pnpm lockfile is missing overrides

Using pnpm, when deploying to firebase we have the following error:
```
Scope: all 11 workspace projects
 ERR_PNPM_LOCKFILE_CONFIG_MISMATCH  Cannot proceed with the frozen installation. The current "overrides" configuration doesn't match the value found in the lockfile

Update your lockfile using "pnpm install --no-frozen-lockfile"; Error ID: be874b7a
Build failed with status: FAILURE and message: installing pnpm dependencies: (error ID: 8622936f):
```

The created `pnpm-lock.yaml` does not contain the `overrides` section from the original `pnpm-lock.yaml`

Note: we are migrating our project from `npm` to `pnpm`. Isolate works correctly when using `npm`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "isolate-package",
-  "version": "1.16.0",
+  "version": "1.17.0-0",
   "description": "Isolate a monorepo package with its shared dependencies to form a self-contained directory, compatible with Firebase deploy",
   "author": "Thijs Koerselman",
   "license": "MIT",

--- a/src/lib/lockfile/helpers/generate-pnpm-lockfile.ts
+++ b/src/lib/lockfile/helpers/generate-pnpm-lockfile.ts
@@ -145,6 +145,11 @@ export async function generatePnpmLockfile({
       ? await pruneLockfile_v9(lockfile, targetPackageManifest, ".")
       : await pruneLockfile_v8(lockfile, targetPackageManifest, ".");
 
+    /** Pruning seems to remove the overrides from the lockfile */
+    if (lockfile.overrides) {
+      prunedLockfile.overrides = lockfile.overrides;
+    }
+
     /**
      * Don't know how to map the patched dependencies yet, so we just include
      * them but I don't think it would work like this. The important thing for

